test/flops.h

```diff
@@ -102,6 +102,61 @@ static double  flops_dgemm(double m, double n, double k)
 static double  flops_sgemm(double m, double n, double k)
     { return    fmuls_gemm(m, n, k) +    fadds_gemm(m, n, k); }
 
+//------------------------------------------------------------ gbmm
```

Comment:
PLASMA has flops macros? Doesn't it? Statically scoped functions are not the best for this

Reply:
Perhaps it does... I do not see them. It looks like everything that calculates flops is a statically scoped function in this file. The only difference is that mine is a good two to three times more complicated to get them done.